Communication/DaemonCommander.cs

```diff
@@ -198,5 +199,49 @@ private static async Task<string> getResponse(string cmd, int timeout)
 
             return body;
         }
-	}
+
+        private static async Task<string> deleteResponse(string cmd, string content, int timeout)
+        {
+
+            var httpClient = new NamedPipeHttpClientBuilder(_daemonHTTPNamedPipe)
+                .WithPerRequestTimeout(TimeSpan.FromSeconds(timeout))
+                .Build();
+
+            var httpContent = new StringContent(content, Encoding.UTF8, "application/json");
+            string command = String.Format("{0}/{1}", _apiPath, cmd);
+            string body = String.Empty;
+
+            HttpRequestMessage request = new HttpRequestMessage

```

Comment:
might wanna generalize this and remove duplication;

we can use a `request` that gets passed into a more generic `sendRequest` function, for each `sendGetRequest`, `sendPostRequest`, and `sendGetRequest` (also getting rid of the `Response` in the name) ...